src/bootstrap/native.rs

```diff
@@ -416,13 +416,18 @@ impl Step for Llvm {
             //        should use llvm-tblgen from there, also should verify that it
             //        actually exists most of the time in normal installs of LLVM.
             let host_bin = builder.llvm_out(builder.config.build).join("bin");
-            cfg.define("CMAKE_CROSSCOMPILING", "True");
             cfg.define("LLVM_TABLEGEN", host_bin.join("llvm-tblgen").with_extension(EXE_EXTENSION));
-            cfg.define("LLVM_NM", host_bin.join("llvm-nm").with_extension(EXE_EXTENSION));
```

Comment:
why was LLVM_NM removed?

Reply:
When cross compiling LLVM, it will show an warning that variable `LLVM_NM` is unused. https://llvm.org/docs/HowToCrossCompileLLVM.html no longer shows `LLVM_NM` as required.

Reply:
Hmm, ok. Do you know in what version the requirement was removed? We currently support 12 and higher: https://github.com/rust-lang/rust/blob/2966c3d81f24ce74b96d26e4e9535677ea3c4e32/src/bootstrap/native.rs#L492

Reply:
Hi @topjohnwu, I don't think I got an answer - do you know when LLVM_NM stopped being required? If it's later than 12 we need to keep it around to support that earlier version.

Reply:
The only place `LLVM_NM` is used is in [this file](https://github.com/llvm/llvm-project/blob/main/llvm/tools/llvm-shlib/CMakeLists.txt) for building `libLLVM-c` on MSVC. I don't think `libLLVM-c` is used by `rustc` at all. I also checked and there are no additional usages ever since it was introduced back in this commit (https://github.com/llvm/llvm-project/commit/1d8cb099231a79b6ad96e745c2d17cf307bea857).

Reply:
awesome, thanks!